scripts/imitation_learning/locomanipulation_sdg/README.md

```diff
@@ -0,0 +1,30 @@
+# Locomanipulation SDG
```

Comment:
Is it possible to move this README to the docs instead of here? Or is there a reason this is here?